### PR TITLE
Remove Update Code from Adguard Home

### PR DESCRIPTION
**Pull request by MickLesk**
## Description

Remove Update Part in Adguad Home, its an fully webui control update

Fixes # (issue)

## Type of change
Please check the relevant option(s):

- [x] Bug fix (non-breaking change that resolves an issue)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [x] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [x] Testing performed (I have tested my changes, ensuring everything works as expected)
- [x] Documentation updated (I have updated any relevant documentation)

## Related Pull Requests / Discussions
- Related Issue: https://github.com/community-scripts/ProxmoxVE/issues/198

### CODE REVIEW

**Review by Mellowlynx — DISMISSED** [review]

File: ct/adguard.sh [hunk 1]
```diff
@@ -60,28 +60,7 @@ if (( $(df /boot | awk 'NR==2{gsub("%","",$5); print $5}') > 80 )); then
...
-msg_info "Starting AdguardHome"
-systemctl start AdGuardHome
-msg_ok "Started AdguardHome"
-
-msg_info "Cleaning Up"
-rm -rf AdGuardHome_linux_amd64.tar.gz AdGuardHome adguard-backup
-msg_ok "Cleaned"
-msg_ok "Updated Successfully"
-exit
+echo "Adguard Home should be updated via the user interface! You can access it at: ${BL}http://${IP}:3000"
```
Comment (havardthom) [inline ct/adguard.sh:63]:
```suggestion
echo "Adguard Home should be updated via the user interface. You can access it at: ${BL}http://${IP}:3000"
```
Can also remove the storage check above?

Comment (MickLesk) [reply ct/adguard.sh:63]: Yes, we could, but personally I would leave it in for informational reasons, or what do you think?

Comment (havardthom) [reply ct/adguard.sh]: Sure

**Review by havardthom — DISMISSED** [review]

File: json/adguard.json [hunk 2]
```diff
@@ -30,5 +30,10 @@
         "username": null,
         "password": null
     },
-    "notes": []
-}
\ No newline at end of file
+    "notes": [
+        {
+            "text": "AdguardHome is only updatable via the WebUI!",
+            "type": "info"
```
Comment (havardthom) [inline json/adguard.json:36]:
```suggestion
            "type": "warning"
```

File: json/adguard.json [hunk 3]
```diff
@@ -30,5 +30,10 @@
         "username": null,
         "password": null
     },
-    "notes": []
-}
\ No newline at end of file
+    "notes": [
+        {
+            "text": "AdguardHome is only updatable via the WebUI!",
```
Comment (havardthom) [inline json/adguard.json:35]:
```suggestion
            "text": "AdguardHome is only updatable via the WebUI!",
```
Think having this extra note in documentation is quite verbose, is it not enough to get the info when running update?

**Comment by MickLesk** [comment]
@havardthom we should discuss later to have an Info flag in Blue or something?

File: json/adguard.json (hunk 3 above)
Comment (havardthom) [reply json/adguard.json:35]: If we do want it I would change it to similar message as above: `Adguard Home should be updated via the user interface`

**Review by havardthom — APPROVED** [review]

**Review by Mellowlynx — APPROVED** [review]

**Pull request merged**